commons/src/main/java/org/archive/util/BloomFilter64bit.java

```diff
@@ -141,45 +80,18 @@ public BloomFilter64bit( final long n, final int d, boolean roundUp) {
      * @param roundUp if true, round bit size up to next-nearest-power-of-2
      */
     public BloomFilter64bit(final long n, final int d, Random weightsGenerator, boolean roundUp ) {
+        delegate = com.google.common.hash.BloomFilter.create(Funnels.unencodedCharsFunnel(), Ints.saturatedCast(n), 0.0000003);
```

Comment:
For sure. When I searched through the code I did not see any varying parameters being passed. The only use I found was this: https://github.com/internetarchive/heritrix3/blob/master/engine/src/main/java/org/archive/crawler/util/BloomUriUniqFilter.java#L89

Reply:
Ah! I understand! This would usually be called from the Spring crawler beans rather than directly in the code, and those beans files are not in the core H3 repository.

Reply:
https://github.com/internetarchive/heritrix3/pull/314